064_reverseList.js

```diff
@@ -0,0 +1,11 @@
+// 64. Write function that reverses a list, preferably in place.
```

Comment:
List should be reversed, not string. Example [1, 2, 3, 4, 5] should be [5, 4, 3, 2, 1]